src/components/ApplicationFooter.jsx

```diff
@@ -3,17 +3,127 @@ import {
   Box,
   Grid,
   Typography,
-  useMediaQuery,
-  useTheme,
+  CardMedia,
+  makeStyles,
 } from "@material-ui/core";
 import { Link } from "react-router-dom";
 import LogoCHWL from "../assets/LogoCHWL.png";
 import { useSelector } from "react-redux";
 
+const useStyles = makeStyles((theme) => ({
+  footerContainer: {
+    [theme.breakpoints.up("xs")]: {
+      display: "flex",
+      flexDirection: "column",
+      width: "90%",
+      maxWidth: "800px",
+      marginLeft: "auto",
+      marginRight: "auto",
+      position: "relative",
+      bottom: "0",
+    },
+    [theme.breakpoints.up("lg")]: {
+      maxWidth: "90%",
+      marginLeft: "10%",
+      marginRight: "5%",
+    },
+  },
+  gridItemWithDivider: {
+    [theme.breakpoints.up("xs")]: {
+      display: "flex",
+      alignItems: "center",
+      alignContent: "center",
+      justifyContent: "center",
+      textAlign: "center",
+      padding: "12px 24px",
+      height: "180px",
+      borderBottom: "1px solid rgba(0,0,0,0.3)",
+    },
+    [theme.breakpoints.up("lg")]: {
+      borderBottom: "none",
+      borderRight: "1px solid rgba(0,0,0,0.3)",
+    },
+  },
+  gridItemNoDivider: {
+    [theme.breakpoints.up("xs")]: {
+      display: "flex",
+      alignItems: "center",
+      alignContent: "center",
+      justifyContent: "center",
+      textAlign: "center",
+      padding: "12px 24px",
+      height: "180px",
+    },
+    [theme.breakpoints.up("lg")]: {
+      justifyContent: "flex-start",
+    },
+  },
+  hiddenScrollContainer: {
+    [theme.breakpoints.up("xs")]: {
+      width: "100%",
+      height: "100%",
+      marginTop: "20px",
+      marginBottom: "20px",
+      paddingTop: "10px",
+      textAlign: "center",
+      overflow: "hidden",
+      position: "relative",
+    },
+    [theme.breakpoints.up("sm")]: {
+      marginTop: "40px",
+      marginBottom: "0px",
+      paddingTop: "20px",
+    },
+    [theme.breakpoints.up("lg")]: {
+      marginTop: "0px",
+      marginBottom: "0px",
+      paddingTop: "20px",
+    },
+  },
+  hiddenScrollText: {
+    [theme.breakpoints.up("xs")]: {
+      top: "0",
+      left: "0",
+      bottom: "-40px",
+      right: "-20px",
```

Comment:
that is interesting.. why negative?

Reply:
Its to hide the scroll bars that appears because of the overflow and text wrap